src/R/Components/Test/PackageManager/PackageManagerIntegrationTest.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition.Hosting;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.R.Components.InteractiveWorkflow;
+using Microsoft.R.Components.Test.Fakes.InteractiveWindow;
+using Microsoft.R.Host.Client;
+using Microsoft.R.Host.Client.Test.Script;
+using Microsoft.R.Support.Settings;
+using Microsoft.UnitTests.Core.XUnit;
+
+namespace Microsoft.R.Components.Test.PackageManager {
+    public class PackageManagerIntegrationTest : IDisposable {
```

Comment:
You can implement `IAsyncLifetime` and move `CreateWorkflowAsync` body into `InitializeAsync`